ReactNativeClient/lib/database-driver-react-native.js

```
@@ -50,6 +50,12 @@ class DatabaseDriverReactNative {
 		});
 	}
 
+	loadExtension(path) {
+		return new Promise(() => {
+			throw new Error(`No extension support for ${path} in react-native-sqlite-storage`);
+		});
```

no need to wrap in new Promise - you can simply throw the exception